src/routes/api/index.js

```diff
@@ -1,11 +1,13 @@
-const router = require("express").Router();
+import express from 'express';
+
+const router = express.Router();
 
 router.use("/", require("./users"));
```

Comment:
```suggestion
router.use("/", "users"));
```
This must have been imported earlier to be used as suggested in the above way
but you cant use the `require`,  it is es5 ... remove until the routes has been created.
**OR** Just comment out for now